src/components/PingDescription.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
+  import marked from "marked";
+
+  export let app;
+  export let name = "";
+  export let description;
+</script>
+
+<a href={`/apps/${app}/pings/${name}`}>{name}</a>
```

Comment:
Should probably not include this by default, since in some cases (on the ping page) we may not show this.